lib/modules/manager/ansible-galaxy/extract.spec.ts

```diff
@@ -1,9 +1,14 @@
+import { codeBlock } from 'common-tags';
 import { Fixtures } from '../../../../test/fixtures';
 import { getSliceEndNumber } from './extract';
 import { extractPackageFile } from './';
 
 const yamlFile1 = Fixtures.get('requirements01.yml');
 const yamlFile2 = Fixtures.get('requirements02.yml');
+const yamlFile3 = codeBlock`collections:
+- name: https://github.com/lowlydba/lowlydba.sqlserver.git
+  type: git 
+  version: 1.1.3`;
```

Comment:
move to test

Reply:
Does that mean I should change the test case?

Reply:
inline to test case instead of defining on module scope.